.github/vars/main.yaml

```diff
@@ -9,6 +9,9 @@ molecule_images:
   - name: fedora-31
     image: fedora:31
     command: /usr/sbin/init
```

Comment:
Are we still doing non-upgrade tests against it?

Reply:
removed, as no testing on f31